### PR TITLE
Changed the restore namespace

### PR DESCRIPTION
**Pull request by skonda-px**
**What this PR does / why we need it**:
Changed the restore namespace name.

**Which issue(s) this PR fixes** (optional)
Closes #PA-619

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by skonda-px** [comment]
Aetos link: http://aetos.pwx.purestorage.com/resultSet/testSetID/104949

File: drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
       maxSurge: 1
       maxUnavailable: 1
     type: RollingUpdate
-  replicas: 2
+  replicas: 1
```
Comment (mkoppal-px) [inline drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml:11]: Please revert this

File: tests/backup/backup_test.go [hunk 2]
```diff
@@ -6488,7 +6488,7 @@ var _ = Describe("{IssueMultipleDeletesForSharedBackup}", func() {
 				backupMap[backupName] = backupUID
 
 				// Start Restore
-				namespaceMapping[bkpNamespaces[0]] = bkpNamespaces[0] + user
+				namespaceMapping[bkpNamespaces[0]] = user
```
Comment (mkoppal-px) [inline tests/backup/backup_test.go:6491]: Can't we just use `bkpNamespaces[0] + "restored"` as the namespace at the destination cluster while restoring?

Comment (skonda-px) [reply tests/backup/backup_test.go]: Done

File: drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml (hunk 1 above)
Comment (skonda-px) [reply drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml]: done

**Comment by skonda-px** [comment]
Aetos link: http://aetos.pwx.purestorage.com/resultSet/testSetID/105081

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**